qa/automation/Starynina.Scenarios/3. Open job positions page/3.1.List of open job positions.feature

```diff
@@ -0,0 +1,18 @@
+Feature: 3.1. List of open job positions
+
+Background:
+	Given user is on the https://career.quantori.com/ru/positions page
+	And there is the list of open job positions
+
+Scenario: 3.1_01 User can choose one of job positions
+	When user places the cursor on one of the positions
+	Then the selected line is highlighted in blue
```

Comment:
Checking the backlight of the button is not part of the functionality it is a user interface
I think it should be deleted
![Screenshot_2](https://user-images.githubusercontent.com/95017779/144242098-5f898460-5a5b-48c1-943b-b9ee8d720305.png)
![Screenshot_1](https://user-images.githubusercontent.com/95017779/144242100-4a0ae418-4aa6-45e1-a32d-250d2fde6f82.png)